charts/rstudio-connect/values.yaml

```
@@ -25,6 +25,9 @@ sharedStorage:
   requests:
     # -- The volume of storage to request for this persistent volume claim
     storage: "10Gi"
+  # -- Define the annotations for the PVC resource
```

Suggested docstring, here and elsewhere:

```suggestion
  # -- Annotations for the PersistentVolumeClaim
```